ui/src/app/views/resource-card/component.ts

```diff
@@ -154,28 +154,20 @@ export class ResourceCardComponent implements OnInit {
     switch (this.resourceType) {
       case ResourceType.NOTEBOOK: {
         this.workspacesService.deleteNotebook(this.wsNamespace, this.wsId, $event.name)
-          .subscribe(() => {
-            this.onUpdate.emit();
-            this.deleteModal.close();
-          });
+          .subscribe(() => this.onUpdate.emit());
         break;
       }
       case ResourceType.COHORT: {
         this.cohortsService.deleteCohort(this.wsNamespace, this.wsId, $event.id)
-          .subscribe(() => {
-            this.onUpdate.emit();
-            this.deleteModal.close();
-          });
+          .subscribe(() => this.onUpdate.emit());
         break;
       }
       case ResourceType.CONCEPT_SET: {
         this.conceptSetsService.deleteConceptSet(this.wsNamespace, this.wsId, $event.id)
-          .subscribe(() => {
-            this.onUpdate.emit();
-            this.deleteModal.close();
-          });
+          .subscribe(() => this.onUpdate.emit());
       }
     }
+    this.deleteModal.close();
```

Comment:
Was this change intentional? Looks like it may have reverted something that Bru did